python/cugraph/tests/test_traveling_salesman.py

```diff
@@ -0,0 +1,73 @@
+# Copyright (c) 2019-2020, NVIDIA CORPORATION.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import gc
+
+import pytest
+
+import cugraph
+from cugraph.tests import utils
+import rmm
+import cudf
+import time
+import numpy as np
+
+TSPLIB_REF = "../datasets/ref/tsp/tsplib.csv"
```

Comment:
We've made a change to support decoupling tests from the source tree, which means we're not hardcoding relative paths anymore.  There's an example [here](https://github.com/rapidsai/cugraph/pull/1337/files#diff-d05ca511f68d1d1443c23b813f82c300d4ede3aeb6ef5b8edc02174f149b43bfR33) in PR #1337

Reply:
Removed since the ref score is now in the dataset tuple: https://github.com/rapidsai/cugraph/pull/1360/commits/923bd9ef94bf6fb9dee55f344379a5485c68a830